packages/deployer/tasks/deploy.js

```diff
@@ -1,17 +1,6 @@
 const { task } = require('hardhat/config');
 const { TASK_COMPILE } = require('hardhat/builtin-tasks/task-names');
 
-// This fixes a very strange bug in hardhat that should be fixed soon.
-// When using the hardhat network, the signer suddenly runs out of ETH =O
-// This is caused by a prototype pollution problem in ANTLR in Immutable.js.
-// It should be fixed soon in a future hardhat version.
-// If fixes, you should be able to run `npx hardhat deploy --no-confirm --clear --network hardhat`
-// without errors.
-// On the meantime, this line produces the local fix.
-// Repo used to reproduce: https://github.com/ajsantander/weird
-// PR that fixes this in hardhat: https://github.com/nomiclabs/hardhat/pull/1945
-require('@solidity-parser/parser');
-
```

Comment:
This above is the workaround that was removed. It is no longer needed. with the fixes within hardhat 2.6.6.